ledger-service/http-json/src/itlib/scala/http/AbstractWebsocketServiceIntegrationTest.scala

```diff
@@ -1147,7 +1147,7 @@ abstract class AbstractWebsocketServiceIntegrationTest(val integration: String)
 
   // Following #16782, we use canton community edition over sandbox-on-x.
   // Pruning is only enabled on enterprise edition, TODO: #16832
-  "fail reading from a pruned offset" ignore withHttpService { fixture =>
+  "fail reading from a pruned offset" in withHttpService { fixture =>
```

Comment:
This turns a pruning test back on, that had been ignored when we switched from sandbox-on-x to canton. This is the test which now fails, even with canton EE